HotelApi/Controllers/ReservationRequest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace HotelApi.Controllers
+{
+    [DataContract]
+    public class ReservationRequest
+    {
+        [DataMember(Name = "hotel_id")]
```

Comment:
Nice to have.

Using DataContract attributes you can specify if field is mandatory or not using IsRequired

Reply:
applied